test_scripts/RC/MultipleModules/ModulesAllocation/060_GetInteriorVehicleDataConsent_all_correct_one_module_rejected.lua

```diff
@@ -0,0 +1,92 @@
+--     (moduleType: <moduleType>, allowed: [false])
+--   Check:
+--    SDL sends RC.GetInteriorVehicleDataConsent request to HMI with (moduleType: <moduleType>, moduleIds:[<moduleId>])
+--    SDL responds on GetInteriorVehicleDataConsent RPC with resultCode:"SUCCESS", success:true
+--    SDL does not send OnRCStatus notifications to HMI and Apps
+-- 3) Try to reallocate module (moduleType: <moduleType>, moduleId: <moduleId>)
+--     to App1 via SetInteriorVehicleData RPC sequentially
+--   Check:
+--    SDL does not send GetInteriorVehicleDataConsent RPC to HMI
+--    SDL rejects allocation of module (moduleType: <moduleType>, moduleId: <moduleId>) to App1
+--     and does not send OnRCStatus notifications to HMI and Apps
+--    SDL responds on SetInteriorVehicleData RPC with resultCode: REJECTED
+---------------------------------------------------------------------------------------------------
+local runner = require('user_modules/script_runner')
+local common = require("test_scripts/RC/MultipleModules/commonRCMulModules")
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+local appLocation = {
+  [1] = common.grid.BACK_RIGHT_PASSENGER,
+  [2] = common.grid.BACK_CENTER_PASSENGER
+}
+
+local testServiceArea = common.grid.BACK_SEATS
+
+local rcAppIds = { 1, 2 }
+
+local rcCapabilities = common.initHmiRcCapabilitiesForRelease(testServiceArea)
+local testModulesArray = common.buildTestModulesArrayFirst(rcCapabilities)
+
+--[[ Scenario ]]
+runner.Title("Preconditions")
+runner.Step("Clean environment", common.preconditions)
+runner.Step("Prepare preloaded policy table", common.preparePreloadedPT, { rcAppIds })
+runner.Step("Prepare RC modules capabilities and initial modules data", common.initHmiDataState, { rcCapabilities })
+runner.Step("Start SDL, HMI, connect Mobile, start Session", common.start, { rcCapabilities })
+runner.Step("Set RA mode: ASK_DRIVER", common.defineRAMode, { true, "ASK_DRIVER" })
+runner.Step("Register App1", common.registerAppWOPTU, { 1 })
+runner.Step("Register App2", common.registerAppWOPTU, { 2 })
+runner.Step("Activate App1", common.activateApp, { 1 })
+runner.Step("Send user location of App1 (Back Seat)", common.setUserLocation, { 1, appLocation[1] })
+runner.Step("Activate App2", common.activateApp, { 2 })
+runner.Step("Send user location of App2 (Back seat)", common.setUserLocation, { 2, appLocation[2] })
+
+for _, testModule in ipairs(testModulesArray) do
+  runner.Step("Allocate module [" .. testModule.moduleType .. ":" .. testModule.moduleId .. "] to App2",
+      common.rpcSuccess, { testModule.moduleType, testModule.moduleId, 2, "SetInteriorVehicleData" })
+end
+
+runner.Title("Test")
+runner.Step("Activate App1", common.activateApp, { 1 })
+for _, testModule in ipairs(testModulesArray) do
+  runner.Step("Disallow module [" .. testModule.moduleType .. ":" .. testModule.moduleId
```

Comment:
This step does get negative consent from driver despite saying `without asking driver`.